src/Core/Content/Flow/Dispatching/AbstractFlowLoader.php

```diff
@@ -4,9 +4,6 @@
 
 use Shopware\Core\Framework\Log\Package;
 
-/**
- * @internal not intended for decoration or replacement
```

Comment:
what changed here that we now think it is a good idea to make the API public?

There are enough extension points in the flow builder itself, so what is the use case for this needing to be public?